proxyclient/m1n1/proxyutils.py

```diff
@@ -185,6 +191,8 @@ def push_adt(self):
         self.iface.writemem(adt_base, self.adt_data)
 
     def disassemble_at(self, start, size, pc=None):
+        '''Start, len, [pc] - disassemble len bytes of memory from start
```

Comment:
Same as above, why not show the function call syntax?

Reply:
Fixed in latest commit